src/Posts/PostList.jsx

```diff
@@ -0,0 +1,15 @@
+import PostListItem from "./PostListItem"
+
+function PostList(props) {
+  const {  posts, contacts } = props
+
+  return (
+    <ul className="post-list">
+      {posts.map((post, index) => (
```

Comment:
You're using the array index as a key for each PostListItem. While this is fine for static lists that don't change, if your list can change (items being added, removed, or reordered), using index as a key can lead to performance issues and bugs with component state. If your posts have unique identifiers, use those instead

Reply:
Resolved!